packages/web/src/pages/search-page-v2/utils.tsx

```diff
@@ -0,0 +1,125 @@
+import { Mood } from '@audius/sdk'
+
+type MoodInfo = {
+  label: Mood
+  value: Mood
+  icon: JSX.Element
+}
+
+export const MOODS: Record<Mood, MoodInfo> = {
```

Comment:
nice just double checking why we cant use the other instance of this? slightly different or something?

Reply:
it could be cleaned up for sure. the other instance have the icon after the text instead of before so did this to match the designs, but I might update the other instance to look like this and just update all of the references.

Reply:
I'll write that down for another PR